### PR TITLE
CameraServoDeactivation

### PR DESCRIPTION
**Pull request by IsabelleTornqvist**
Deactivation/activation of camera servos X and Y. Refined temperature code.

### CODE REVIEW

File: SmartCar/Arduino/sketch_main/sketch_main.ino [hunk 1]
```diff
@@ -277,9 +279,17 @@ void command() {
           ultrasonic_activation = all_activation;
           temperature_activation = all_activation;
           flame_activation = all_activation;        
-          break;      
+          break;     
+
+        case 'X':
```
Comment (pontuslae) [inline SmartCar/Arduino/sketch_main/sketch_main.ino]: Add comment here

File: SmartCar/Arduino/sketch_main/sketch_main.ino [hunk 2]
```diff
@@ -277,9 +279,17 @@ void command() {
...
+          camera_servo_activation_horizontal = !camera_servo_activation_horizontal;
+          break;
 
-           // Camera servos,
+        case 'Y':
```
Comment (pontuslae) [inline SmartCar/Arduino/sketch_main/sketch_main.ino]: Add comment here

File: SmartCar/Arduino/sketch_main/sketch_main.ino [hunk 3]
```diff
@@ -131,20 +133,20 @@ void setup() {
...
+  }else if( temperature < 100){
+    tempString += "0";
   }
+  tempString += temperature; 
 
 //delay for sending the temperature: if the temperature has changed, send the temperature 
   if(temperature_old != temperature){
-      Serial.println(tempString);   
+      Serial.println(tempString);
+      //Serial.println("t");
```
Comment (pontuslae) [inline SmartCar/Arduino/sketch_main/sketch_main.ino]: unnecessary. Remove this

**Review by timjon — CHANGES_REQUESTED** [review]
No comment added for new version and what has been edited

File: SmartCar/Arduino/sketch_main/sketch_main.ino [hunk 4]
```diff
@@ -54,6 +58,8 @@ boolean ultrasonic_activation = true;
 boolean servo_activation = true;     
 boolean motor_activation = true;
 boolean all_activation = true; //Used for de-_activation of all sensors/modules 
+boolean camera_servo_activation_horizontal = true; //X is horizontal  
+boolean camera_servo_activation_vertical = true; //Y is vertical
```
Comment (timjon) [inline SmartCar/Arduino/sketch_main/sketch_main.ino]: Should be "camera_servo_horizontal_activation" and "camera_servo_vertical_activation for consistency"

File: SmartCar/Arduino/sketch_main/sketch_main.ino [hunk 5]
```diff
@@ -131,20 +137,19 @@ void setup() {
 void printTemperature(DeviceAddress deviceAddress){  
   temperature_old = temperature;
   temperature = (int)temperature_sensor.getTempC(deviceAddress);
-  String tempString;
+  String tempString = "t";
```
Comment (timjon) [inline SmartCar/Arduino/sketch_main/sketch_main.ino]: temp_string maybe?

**Review by timjon — CHANGES_REQUESTED** [review]

**Review by timjon — APPROVED** [review]

**Review by pontuslae — APPROVED** [review]
Good job

**Review by ElaineQvarnstrom — APPROVED** [review]
Looking good!

**Pull request merged**